datahub-web-react/src/app/shared/tags/AddTagsTermsModal.tsx

```diff
@@ -351,7 +384,18 @@ export default function EditTagTermsModal({
             })
             .catch((e) => {
                 message.destroy();
-                message.error({ content: `Failed to remove: \n ${e.message || ''}`, duration: 3 });
+                if (
+                    resources.length > 1 &&
+                    e.message === 'Unauthorized to perform this action. Please contact your DataHub administrator.'
```

Comment:
Generally, we try to avoid depending on the error message itself. There should also be a status code included in the DataHub error message. If its unauthorized, it will return 401 in cases of unauthorized. 

It's okay to handle these errors in a more specific way, but I'd recommend that we 

1. Break out this logic into a shared handleBatchError method which can be reused across these components.
2. Avoid using overly specific error message language (in this case, "datasets", since Tags and Terms can be added to many entity types beyond just datasets

Cheers
John